cherrypy/test/test_httputil.py

```diff
@@ -30,3 +31,17 @@ def test_urljoin(self, script_name, path_info, expected_url):
         """Test all slash+atom combinations for SCRIPT_NAME and PATH_INFO."""
         actual_url = httputil.urljoin(script_name, path_info)
         assert actual_url == expected_url
+
+
+def test_valid_status():
```

Comment:
Would you please move this to class or remove the class above so that all tests would be in the same style?